### PR TITLE
KIALI-2835 Document config write operations

### PR DESCRIPTION
**Pull request by lucasponce**
We had no documentation in the web for write operations and wizards.

This is not fully completed but I guess it helps if it needs to be used as reference.

### CODE REVIEW

File: content/documentation/overview/index.adoc [hunk 1]
```diff
@@ -104,7 +104,11 @@ The Istio Config menu entry displays a list of all of the available Istio config
 .List of Istio Configs
 image::istio-list.png[Demo Website]
 {nbsp} +
-You can inspect the configuration yaml around a specific Istio object.
+You can view and edit and delete the configuration yaml around a specific Istio object.
```
Comment (xeviknal) [inline content/documentation/overview/index.adoc]: what about "view, edit and delete"?

File: content/documentation/overview/index.adoc [hunk 2]
```diff
@@ -104,7 +104,11 @@ The Istio Config menu entry displays a list of all of the available Istio config
...
+{nbsp} +
```
Comment (aljesusg) [inline content/documentation/overview/index.adoc]: Remove and by ,
`You can view and edit and delete the configuration yaml around a specific Istio object.`
by
`You can view , edit and delete the configuration yaml around a specific Istio object.`

File: content/documentation/overview/index.adoc [hunk 3]
```diff
@@ -238,6 +242,70 @@ This section lists all the validations that Kiali performs on all Istio configur
...
+=== Istio Wizards
+
+Kiali provides different actions to create, update and delete Istio configuration driven by Wizards.
+These actions are located under Service Details page.
+
+.Service Details Actions
+image::service-istio-actions.png[Service Istio Actions]
+{nbsp} +
+These actions are enabled by default. +
+Kiali can be also installed in "view only" mode to restrict any write operation on Istio configuration. +
```
Comment (xeviknal) [inline content/documentation/overview/index.adoc]: Not sure about this, but it sounds more correct to me the following:
'Kiali can be also installed' -> 'Kiali can also be installed'

File: content/documentation/overview/index.adoc [hunk 4]
```diff
@@ -238,6 +242,70 @@ This section lists all the validations that Kiali performs on all Istio configur
...
+Check https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml#L134[Kiali Operator CR] to get more details about how to configure this option.
```
Comment (aljesusg) [inline content/documentation/overview/index.adoc]: Please add option window="_blank" to open this in a new tab
`[Kiali Operator CR, window="_blank"]`

File: content/documentation/overview/index.adoc [hunk 5]
```diff
@@ -238,6 +242,70 @@ This section lists all the validations that Kiali performs on all Istio configur
...
+
+.Advanced options section
+image::wizard-advanced-options.png[Advanced Options]
+{nbsp} +
+When mTLS is enabled by default in the global cluster or namespace this option is already preselected.
+
+==== More Wizard examples
+
+The following article https://medium.com/kialiproject/kiali-observability-in-action-for-istio-service-mesh-69127f792103[Kiali: Observability in Action for Istio Service Mesh]
+describes more examples of how to use the Kiali Wizards to configure Istio configuration.
```
Comment (aljesusg) [inline content/documentation/overview/index.adoc]: The same plz add option window _blank

File: content/documentation/overview/index.adoc (hunk 4 above)
Comment (xeviknal) [inline content/documentation/overview/index.adoc]: What do you thinking of joining this sentence with the previous one? I feel that both sentences are talking about the same. Not blocking for sure.

**Comment by lucasponce** [comment]
@xeviknal @aljesusg comments addressed.
Thanks for the review.

**Pull request merged**